test/evmc/mocked_host.hpp

```diff
@@ -63,6 +63,9 @@ struct MockedAccount
     /// The account storage map.
     std::map<bytes32, StorageValue> storage;
 
+    /// The account transient storage.
+    std::unordered_map<bytes32, bytes32> transient_storage;
```

Comment:
```suggestion
    std::map<bytes32, bytes32> transient_storage;
```

Same reasoning as for ``storage`` applies here.

Reply:
Did not change this on purpose - we don't have support for it in `EVMHost.cpp`, so I feel like this should be done in the PR that actually introduces this change (this is all for fuzzing anyway).

Reply:
We're also planning to revert this to the original state, remember? I.e. perform the `unordered_map` -> `map` conversion on the user side (`EVMHost`), so we can introduce `evmc` as a submodule.